test/remote/gateways/remote_global_collect_test.rb

```diff
@@ -12,6 +12,28 @@ def setup
     @cabal_card = credit_card('6271701225979642', brand: 'cabal')
     @declined_card = credit_card('5424180279791732')
     @preprod_card = credit_card('4111111111111111')
+    @apple_pay = network_tokenization_credit_card('4567350000427977',
+      payment_cryptogram: 'EHuWW9PiBkWvqE5juRwDzAUFBAk=',
+      month: 10,
+      year: 24,
+      first_name: 'John',
+      last_name: 'Smith',
+      verification_value: '737',
+      source: :apple_pay)
+
+    @google_pay = network_tokenization_credit_card('4567350000427977',
+      payment_cryptogram: 'EHuWW9PiBkWvqE5juRwDzAUFBAk=',
+      month: '01',
+      year: Time.new.year + 2,
+      source: :google_pay,
+      transaction_id: '123456789',
+      eci: '05')
+
+    @google_pay_pan_only = network_tokenization_credit_card('4567350000427977',
```

Comment:
❓  Question:

Maybe I'm wrong, but pan_only google pay tokens are going to be passed as instances of CreditCard not NetworkTokenization ?

Reply:
Yes, you are right.